core/src/main/java/org/bitcoinj/params/MainNetParams.java

```diff
@@ -318,7 +318,7 @@ public MainNetParams() {
         deterministicMasternodesEnabled = true;
 
         DIP0008BlockHeight = 1088640;
-        DIP0024BlockHeight = Integer.MAX_VALUE;
+        DIP0024BlockHeight = 1737792 + 5 * 288; // DIP24 activation time + 5 cycles
```

Comment:
now that DIP24 is activated, we can hard code the height here.

There is no longer a need to determine this using other means.